app/components/Home.tsx

```diff
@@ -0,0 +1,368 @@
+import * as React from 'react';
+const { EditorKit, EditorKitDelegate } = require('sn-editor-kit');
+const MarkdownIt = require('markdown-it');
+
+type Mode = {
+  type: ModeType;
+  label: string;
+  css: string;
+};
+
+enum ModeType {
+  Edit = 0,
+  Split = 1,
+  Preview = 2,
+}
+
+enum MouseEvent {
+  Down = 'mousedown',
+  Move = 'mousemove',
+  Up = 'mouseup',
+}
+
+enum HtmlElementId {
+  ColumnResizer = 'column-resizer',
+  Editor = 'editor',
+  EditorContainer = 'editor-container',
+  Header = 'header',
+  Preview = 'preview',
+  SimpleMarkdown = 'simple-markdown',
```

Comment:
This is poorly named (probably pre-existing). We should take this chance to rename this to something that better describes what exactly this element is.

Reply:
How do you want to rename them? We'll probably need to change them in the CSS as well

Reply:
What exactly is the element? If it's a container, it should be called that. If it's the root, it should be called that.

Reply:
What exactly do you want renamed?

Reply:
We can change `simple-markdown` to `root` or `preview` to `preview-container`, but I think `editor-container` and `header` are pretty clear